registry/donor/forms.py

```diff
@@ -37,3 +37,73 @@ def add_one_rodne_cislo(self, rodne_cislo):
 class NoteForm(FlaskForm):
     rodne_cislo = HiddenField(validators=[DataRequired()])
     note = TextAreaField("Poznámka k dárci")
+
+
+class DonorsOverrideForm(FlaskForm):
+    rodne_cislo = StringField("Rodné číslo", validators=[DataRequired()])
+    first_name = StringField("Jméno")
+    last_name = StringField("Příjmení")
+    address = StringField("Adresa")
+    city = StringField("Město")
+    postal_code = StringField("PSČ")
+    kod_pojistovny = StringField("Pojišťovna")
+
+    _fields_ = [
```

Comment:
Pokud je atribut třídy privátní, měl by mít podtržítko jen na začátku názvu:
```suggestion
    _fields = [
```

Reply:
`_fields` už je zabrané třídou FlaskForm (obsahuje něco podobného jako toto, ale je tam navíc i `"csrf_token"`). Ale mohl bych `_fields_` změnit na něco jako `_data_fields` nebo `__fields`.

Reply:
Pak je to správně, protože při kolizi jmen se přidává underscore nakonec - např. u `class_`, takže můj koment ignoruj.